openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx

```diff
@@ -138,13 +137,13 @@ const AssetsTabs = forwardRef(
               [EntityType.PIPELINE]: pipelineResponse.data.hits.total.value,
               [EntityType.MLMODEL]: mlmodelResponse.data.hits.total.value,
               [EntityType.CONTAINER]: containerResponse.data.hits.total.value,
-              [EntityType.GLOSSARY]:
+              [EntityType.GLOSSARY_TERM]:
                 type !== AssetsOfEntity.GLOSSARY
                   ? glossaryResponse.data.hits.total.value
                   : 0,
             };
 
-            setItemCount(counts);
+            setItemCount(counts as Record<EntityType, number>);
```

Comment:
```suggestion
            setItemCount(counts);
```

Reply:
required as there are many EntityType, but we are using 6 or 7